### ISSUE TITLE
`arc logs` on a custom filepath blows up

### ISSUE THREAD

**Issue by filmaj** [architect/architect#1020]
**Describe the issue**
```
16:40:17 in Conduit on  paris [!?] is 📦 v1.0.1 via ⬢ v14.14.0
➜ arc logs dist/ws/default
         App ⌁ conduit
      Region ⌁ ca-central-1
     Profile ⌁ copper
     Version ⌁ Architect 8.3.2
         cwd ⌁ /Users/filmaj/src/Conduit

logs failed! Cannot read property 'endsWith' of null
TypeError: Cannot read property 'endsWith' of null
    at /Users/filmaj/src/Conduit/node_modules/@architect/logs/src/get-logical-id.js:24:17
    at Array.forEach (<anonymous>)
    at /Users/filmaj/src/Conduit/node_modules/@architect/logs/src/get-logical-id.js:23:34
    at Array.forEach (<anonymous>)
    at getLogicalID (/Users/filmaj/src/Conduit/node_modules/@architect/logs/src/get-logical-id.js:21:27)
    at Function.readLogs [as read] (/Users/filmaj/src/Conduit/node_modules/@architect/logs/src/read-logs.js:10:19)
    at logs (/Users/filmaj/src/Conduit/node_modules/@architect/logs/index.js:52:20)
    at Object.cli [as logs] (/Users/filmaj/src/Conduit/node_modules/@architect/logs/cli.js:28:10)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async run (/Users/filmaj/src/Conduit/node_modules/@architect/architect/src/index.js:55:5)
```

Will investigate later but just filing the issue for now.

**Comment by ryanblock** [comment]
Found it! Will be patched in a few.

**Comment by ryanblock** [comment]
Patched in Inventory 1.1.1, which had some other changes that are now live in 8.3.3!
